src/extensions/WorkspaceView/sortingExamples/mountainsort4_example.py

```diff
@@ -0,0 +1,28 @@
+import spikeextractors as se
```

Comment:
MountainSort4 example (not included in the instructions panel yet because I'm not sure if I have the wrapper working properly)